solr/solrj/src/java/org/apache/solr/client/solrj/impl/HttpSolrJdkClient.java

```diff
@@ -64,7 +66,7 @@ public class HttpSolrJdkClient extends HttpSolrClientBase {
   private static final String USER_AGENT =
       "Solr[" + MethodHandles.lookup().lookupClass().getName() + "] 1.0";
 
-  private HttpClient client;
+  protected HttpClient client;
```

Comment:
minor, but can we call this `httpClient` to make it clearer wherever we use it what sort of client we're talking about?

Reply:
I changed the name as you suggested.